README.md

```diff
@@ -333,4 +333,354 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+## Destructuring
+
+  - [5.1](#5.1) <a name='5.1'></a> Use object destructuring when accessing and using multiple properties of an object.
+
+  > Why? Destructuring saves you from creating temporary references for those properties.
+
+    ```javascript
+    // bad
+    function getFullName(user) {
+      const firstName = user.firstName;
+      const lastName = user.lastName;
+
+      return `${firstName} ${lastName}`;
+    }
+
+    // good
+    function getFullName(obj) {
+      const { firstName, lastName } = obj;
+      return `${firstName} ${lastName}`;
+    }
+
+    // best
+    function getFullName({ firstName, lastName }) {
```

Comment:
I'm not sure if I like this one, it's hiding the fact that we are getting an object and it puts the object attributes in the signature of the function. There is a good part tho: since we are not getting the full object as a parameter we minimize the chances to have side effects on the object itself (that is good).

Any thoughts?